merge.py

```diff
@@ -18,6 +18,44 @@
 TMP_DIR_FMT = '/tmp/{dir}'
 
 
+class Command(object):
+    def __init__(self, command, desc, error, shell=False):
+        self.command = command
+        self.desc = desc
+        self.error = error
+        self.shell = shell
+
+
+GIT_CMDS = [
```

Comment:
Comment on this -- something like

```
# Commands to merge a github PR
# Environment variables will be:
# ...
```

So that it is easy to edit/read this.